src/libteam/patch/0003-libteam-Add-fallback-support-for-single-member-port-.patch

```diff
@@ -1,17 +1,16 @@
-From 9b40af58575a89d06be51cfbb5a2265a59826110 Mon Sep 17 00:00:00 2001
-From: yorke <yorke.yuan@asterfusion.com>
-Date: Mon, 3 Jun 2019 12:02:36 +0800
-Subject: [PATCH 3/8] [libteam] Add fallback support for single-member-port LAG
-     From: Haiyang Zheng <haiyang.z@alibaba-inc.com>     Date: Fri, 15 Dec
- 2017 21:07:53 -0800
+commit f475746f56602a40861b8d94eac5f0979c4703f3
+Author: yorke <yorke.yuan@asterfusion.com>
+Date:   Mon Jun 3 12:02:36 2019 +0800
```

Comment:
You can remove it.

Reply:
Thanks for the review. Looks like it's already merged.